base/strings/unicode.jl

```diff
@@ -221,7 +221,7 @@ julia> textwidth('α')
 1
 
 julia> textwidth('❤')
-2
+1
```

Comment:
We should find another example of a fullwidth character.

Reply:
`⏰` ([U+23f0](https://www.fileformat.info/info/unicode/char/23f0/index.htm)) is defined as a wide character by [UAX 11](http://unicode.org/reports/tr11/), so it should always have width 2.

Reply:
The `⛵` (U+2685) in the modified PR is also "wide", so that should be fine too.